lib/Dialog/Dialog.js

```diff
@@ -153,6 +165,12 @@ module.exports = kind(
 	*/
 	create: function () {
 		Popup.prototype.create.apply(this, arguments);
+		if(this.bodyComponents) {
+			this.$.message.destroy();
```

Comment:
Hm, if we removed the definition of the "message" control from our component declaration, would something like this work for the implementation of `initComponents`?

```javascript
initComponents: function () {
			this.inherited(arguments);
			if (this.bodyComponents) {
				var owner = this.hasOwnProperty('bodyComponents') ? this.getInstanceOwner() : this;
				this.$.bodyComps.createComponents(this.bodyComponents,{owner:owner});
			} else {
				this.$.bodyComps.createComponent({
					name: 'message',
					kind: BodyText,
					classes: 'moon-dialog-content'
				});
			}
		}
```